src/tests/multihost/admultidomain/test_multidomain.py

```diff
@@ -0,0 +1,187 @@
+        :customerscenario: True
+        """
+        adchildjoin(membersw='adcli')
+        ad_domain = multihost.ad[1].domainname
+        ad_child_domain = multihost.ad[1].domainname
+
+        # Configure sssd
+        multihost.client[0].service_sssd('stop')
+        client = sssdTools(multihost.client[0], multihost.ad[1])
+        client.backup_sssd_conf()
+        dom_section = f'domain/{client.get_domain_section_name()}'
+        sssd_params = {
+            'ad_domain': ad_child_domain,
+            'debug_level': '9',
+            'use_fully_qualified_names': 'True',
+            'cache_credentials': 'True',
+            'ad_enabled_domains': ad_child_domain
+        }
+        client.sssd_conf(dom_section, sssd_params)
+        client.clear_sssd_cache()
+        # Search for the user in root domain
+        parent_cmd = multihost.client[0].run_command(
+            f"getent passwd user1@%s" % ad_domain,
+            raiseonerr=False
+        )
+        # Search for the user in child domain
+        child_cmd = multihost.client[0].run_command(
+            f"getent passwd child_user1@%s" % ad_child_domain,
+            raiseonerr=False
+        )
+        # Run su command
+        su_result = client.su_success("child_user1@%s" % ad_child_domain)
+
+        client.restore_sssd_conf()
+        client.clear_sssd_cache()
+
+        # Evaluate test results
+        assert parent_cmd.returncode == 2
+        assert child_cmd.returncode == 0
+        assert su_result, "The su command success!"
+
+    @staticmethod
+    def test_0002_bz2018432(multihost, adjoin):
+        """
+        :title: IDM-SSSD-TC: ad_provider: forests:  based SSSD adds more AD
+        domains than it should based on the configuration file
+        :id:
+        :setup:
+          1. Configure several domains, this suite contains 4 trusted domains
+          2. Join client to parent domain
+        :steps:
+          1. Perform sssctl domain-list
+        :expectedresults:
+          1. Only trusted domains listed
+        :customerscenario: True
+        """
+        adjoin(membersw='adcli')
+        ad_domain = multihost.ad[0].domainname
+        ad_child_domain = multihost.ad[1].domainname
+        ad_child1_domain = multihost.ad[2].domainname
+        ad_tree_domain = multihost.ad[3].domainname
+
+        # Configure sssd
+        multihost.client[0].service_sssd('stop')
+        client = sssdTools(multihost.client[0], multihost.ad[0])
+        client.backup_sssd_conf()
+        dom_section = f'domain/{client.get_domain_section_name()}'
+        sssd_params = {
+            'ad_domain': ad_domain,
+            'debug_level': '9',
+            'use_fully_qualified_names': 'True',
+            'cache_credentials': 'True'
+        }
+        client.sssd_conf(dom_section, sssd_params)
+        client.clear_sssd_cache()
+        # List domains
+        domain_list_cmd = multihost.client[0].run_command(f'sssctl domain-list', raiseonerr=False)
```

Comment:
I think you can drop the f from the beginning of the string if there are no variables included in the string.

Reply:
Not sure if you missed a push but, I'm still seeing the f before the single quote.